test/blackbox-tests/test-cases/coq/coqtop-root.t/run.t

```diff
@@ -14,5 +14,5 @@ This test is currently broken due to the workspace resolution being faulty #5899
   1 | (coq.theory
   2 |  (name foo))
   Error: 'coq.theory' is available only when coq is enabled in the dune-project
```

Comment:
```suggestion
  Error: 'coq.theory' is available only when Coq is enabled in the ``dune-project``
```

Reply:
But also technically correct since it is talking about the `coq` mode which is enabled in the `dune-project` file.

Reply:
Since it appears in a command line error message, making it monospace would waste space.

Reply:
@christinerose These are Cram (I/O) tests. The test system runs specified commands, and the output of the commands is tested against what is committed here. Hence, you cannot just change this output because then the tests would fail. If you would want to change this, you'd have to do it where the I/O is actually performed.

(As an additional note, I'd say that the required standard of copy-editing in tests is a bit lower than one would expect for, say, documentation.)

Reply:
Understood. Please feel free to discard anything that's not useful. I was tagged to review, so I did my thing. 😄  .... Do you need me to go back and "approve" it before you can move forward?

Reply:
No, the PR has already been merged and completed. Thanks for your comments @christinerose.